recipe/meta.yaml

```diff
@@ -19,6 +19,8 @@ build:
 
 requirements:
   build:
+    - krb5 1.14.*
+    - libssh2 1.8.*
     # python is a build requirement on Windows to resolve features.
     - python           # [win]
     # perl is required to run the tests on UNIX.
```

Comment:
Because otherwise we will get a matrix of `perl` versions when we re-render. So this is a hack to avoid that problem. More details in issue ( https://github.com/conda-forge/conda-smithy/issues/415 ).